torchvision/prototype/models/depth/stereo/raft_stereo.py

```diff
@@ -462,32 +484,38 @@ def _raft_stereo(
     progress: bool = False,
     shared_encoder_weight: bool = False,
     # Feature encoder
-    feature_encoder_layers: Tuple[int, int, int, int, int],
-    feature_encoder_strides: Tuple[int, int, int, int],
-    feature_encoder_block: Callable[..., nn.Module],
+    feature_encoder_layers: Tuple[int, int, int, int, int] = (64, 64, 96, 128, 256),
+    feature_encoder_strides: Tuple[int, int, int, int] = (1, 1, 2, 2),
+    feature_encoder_block: Callable[..., nn.Module] = ResidualBlock,
     # Context encoder
-    context_encoder_layers: Tuple[int, int, int, int, int],
-    context_encoder_strides: Tuple[int, int, int, int],
+    context_encoder_layers: Tuple[int, int, int, int, int] = (64, 64, 96, 128, 256),
+    context_encoder_strides: Tuple[int, int, int, int] = (1, 1, 2, 2),
+    # if the `out_with_blocks` param of the context_encoder is True, then
+    # the particular output on that level position will have additional `context_encoder_block` layer
     context_encoder_out_with_blocks: List[bool],
-    context_encoder_block: Callable[..., nn.Module],
+    context_encoder_block: Callable[..., nn.Module] = ResidualBlock,
```

Comment:
Not sure how this works. You shouldn't be able to have default value for `context_encoder_block` but no default value for `context_encoder_out_with_blocks` which is before that.

Do we even need the default values here? Typically the internal methods don't have default values (see `_resnet` for example) and we place the weights and process at the end.